plugins/filter_aws/aws.h

```diff
@@ -107,6 +108,9 @@ struct flb_filter_aws {
     flb_sds_t *tag_values;
     /* tag_values_len is an array of lengths related to tag_values items */
     size_t *tag_values_len;
+    /* tag_enabled is an array of bools which define if tag should be injected */
```

Comment:
TODO: update the name of the attribute in the comment, should be `tab_is_enabled`.